source/_integrations/electrasmart.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: Electra Smart
+description: Instructions for how to integrate Electra Air Conditioners within Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.11
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@jafar-atili'
+ha_domain: electrasmart
+ha_platforms:
+  - climate
+
+ha_integration_type: integration
+---
+
+[Electra Air](https://www.electra-air.co.il), is a company which manufactures and sell Air Conditioners.
+
+In order to setup this integration, you must have access to the phone number used to register in the Electra Smart mobile app.
```

Comment:
- Suggestion: `In order to` can be written simpler to as `To`
- Typo: `setup` -> `set up`